python/ray/tune/examples/memnn_example.py

```diff
@@ -0,0 +1,278 @@
+"""Example training a memory neural net on the bAbI dataset.
+
+References Keras and is based off of https://keras.io/examples/babi_memnn/.
+"""
+
+from __future__ import print_function
+
+from keras.models import Sequential, Model, load_model
```

Comment:
oh; as of now, looks like we should move all of this to `tf.keras`; could you do that? should just be an import swap.